src/modules/vendor/mixins/templateMixin.js

```diff
@@ -28,10 +28,7 @@ export const templateMixin = {
     };
   },
   computed: {
-    ...mapState('program', ['card', 'activity']),
```

Comment:
Je ne suis pas sûr qu'il faille supprimer activity ici, ça créer des erreurs dans les cartes (par exemple FillTheGap utilise this.activity.status)

Reply:
outdated